aws-observability-terraform/app-modules/variables.tf

```diff
@@ -155,3 +155,15 @@ variable "rds_monitors_disabled" {
   default     = true
 }
 
+variable "folder_installation_location" {
```

Comment:
Refer comments provided for aws-observability-terraform/variables.tf file change
Update things accordingly